lib/HttpInterface.js

```diff
@@ -46,6 +46,17 @@ function startWebServer(pm2) {
           processes: list
         };
 
+        if (cst.WEB_STRIP_ENV_VARS === true) {
+          for (var i = data.processes.length - 1; i >= 0; i--) {
+            var proc = data.processes[i];
+
+            // Strip important environment variables
+            if (typeof proc.pm2_env === 'undefined' && typeof proc.pm2_env.env === 'undefined') return;
+
```

Comment:
Maybe change 'return' to 'break;

http request will not be ended if return from here.